dakota_manylinux_install_files/build_wheels_gha.sh

```diff
@@ -0,0 +1,166 @@
+
+./bootstrap.sh --with-libraries=python,filesystem,program_options,regex,serialization,system --with-python=$(which python) --with-python-root="$python_root" &> "$INSTALL_DIR/boost_bootstrap.log"
+#./bootstrap.sh --with-libraries=python,filesystem,program_options,regex,serialization,system --with-python-version=$python_version --with-python-root="$python_root"
+sed -i -e "s|.*using python.*|$python_bin_include_lib|" project-config.jam
+echo "# sed -i -e \"s|.*using python.*|$python_bin_include_lib|\" project-config.jam" >> /github/workspace/trace/env
+
+./b2 install -j8 -a cxxflags="-std=c++17" --prefix="$INSTALL_DIR" &> /github/workspace/trace/boost_install.log
+echo "# ./b2 install -j8 -a cxxflags="-std=c++17" --prefix="$INSTALL_DIR" &> /github/workspace/trace/boost_install.log" >> /github/workspace/trace/env
+
+cd $INSTALL_DIR
+DAKOTA_INSTALL_DIR=/tmp/INSTALL_DIR/dakota
+mkdir -p $DAKOTA_INSTALL_DIR
+echo "DAKOTA_INSTALL_DIR=$DAKOTA_INSTALL_DIR" >> /github/workspace/trace/env
+
+wget https://github.com/snl-dakota/dakota/releases/download/v$DAKOTA_VERSION/dakota-$DAKOTA_VERSION-public-src-cli.tar.gz > /dev/null
+tar xf dakota-$DAKOTA_VERSION-public-src-cli.tar.gz
+
+CAROLINA_DIR=/github/workspace
+
+rm -f dakota-$DAKOTA_VERSION-public-src-cli/CMakeLists.txt
+rm -f dakota-$DAKOTA_VERSION-public-src-cli/cmake/DakotaFindPython.cmake
+
+cp $CAROLINA_DIR/dakota_manylinux_install_files/CMakeLists.txt dakota-$DAKOTA_VERSION-public-src-cli/CMakeLists.txt
+cp $CAROLINA_DIR/dakota_manylinux_install_files/DakotaFindPython.cmake dakota-$DAKOTA_VERSION-public-src-cli/cmake
+
+cd dakota-$DAKOTA_VERSION-public-src-cli
+mkdir build
+cd build
+
+export PATH=/tmp/INSTALL_DIR/bin:$PATH
+export PYTHON_INCLUDE_DIRS="$PYTHON_INCLUDE_PATH $PYTHON_DEV_HEADERS_DIR /tmp/INSTALL_DIR/lib"
+export PYTHON_EXECUTABLE=$(which python)
+export LD_LIBRARY_PATH="$INSTALL_DIR/lib:/usr/local/lib:$PYTHON_INCLUDE_PATH:$NUMPY_INCLUDE_PATH:$NUMPY_INCLUDE_PATH/numpy:$PYTHON_DEV_HEADERS_DIR:/tmp/INSTALL_DIR/lib:$LD_LIBRARY_PATH"
+
+echo "export PATH=$PATH" >> /github/workspace/trace/env
+echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH" >> /github/workspace/trace/env
+echo "export PYTHON_INCLUDE_DIRS=$PYTHON_INCLUDE_DIRS" >> /github/workspace/trace/env
+echo "export PYTHON_EXECUTABLE=$PYTHON_EXECUTABLE" >> /github/workspace/trace/env
+
+export BOOST_PYTHON="boost_python$python_version_no_dots"
+export BOOST_ROOT=$INSTALL_DIR
+export PATH="$PATH:$INSTALL_DIR/bin"
+
+# More stable approach: Go via python
+numpy_lib_dir=$(find /tmp/myvenv/ -name numpy.libs)
+export LD_LIBRARY_PATH="/usr/lib:/usr/lib64:$INSTALL_DIR/lib:$INSTALL_DIR/bin:$numpy_lib_dir:$NUMPY_INCLUDE_PATH"
+export CMAKE_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed 's/::/:/g' | sed 's/:/;/g')
+export PYTHON_LIBRARIES="/usr/lib64/"
+export PYTHON_INCLUDE_DIR="/opt/_internal/cpython-3.7.17/include/python3.7m"
```

Comment:
This might cause some issues, no?

Reply:
I think so, though it is only used for the python development headers. This was done before finding the .tar with the headers in the manylinux image, so this might be possible to remove without breaking the build, will try it.